tests/e2e/replication_experimental/multitenancy.py

```diff
@@ -109,7 +106,7 @@ def do_manual_setting_up(connection):
 MEMGRAPH_INSTANCES_DESCRIPTION_WITH_RECOVERY = {
     "replica_1": {
         "args": [
-            "--experimental-enabled=system-replication",
+            "--replication-system-replication-enabled=true",
```

Comment:
I guess this is just not removed yet (it's ok because the PR is still in draft)? But the problem is if it's actually working 😂 In this config, the test should crash and not pass 👀